pkg/tecdsa/signing/states.go

```diff
@@ -41,6 +41,9 @@ const (
 
 	tssRoundNineStateDelayBlocks  = 1
 	tssRoundNineStateActiveBlocks = 5
+
+	finalizationStateDelayBlocks  = 1
+	finalizationStateActiveBlocks = 5
```

Comment:
If no messages are valid in the finalization state, do we need 5 blocks?

Reply:
This is exactly the same case as for DKG, we need to process multiple input messages and produce the result, hence 5 blocks.